src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/WebsocketSimpleScenario.scala

```diff
@@ -0,0 +1,28 @@
+package org.apache.james.gatling.jmap.rfc8621.scenari
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ScenarioBuilder
+import io.gatling.http.Predef._
+import org.apache.james.gatling.control.UserFeeder.UserFeederBuilder
+import org.apache.james.gatling.jmap.rfc8621.JmapMailbox.inboxIdPath
+import org.apache.james.gatling.jmap.rfc8621.JmapWebsocket.{enablePush, getMailboxesWs, websocketConnect}
+import org.apache.james.gatling.jmap.rfc8621.SessionStep
+
+import scala.concurrent.duration._
+
+class WebsocketSimpleScenario {
+  def generate(duration: Duration, userFeeder: UserFeederBuilder): ScenarioBuilder =
+    scenario("WebsocketSimpleScenario")
+      .feed(userFeeder)
+      .exec(SessionStep.retrieveAccountId)
+      .during(duration) {
+        exec(websocketConnect().onConnected(
```

Comment:
Maybe add a delay before enabling the PUSH? (nevern know...)

Reply:
Nothing to do with it :)